numpy/fft/tests/test_helper.py

```diff
@@ -74,5 +75,47 @@ def test_not_last_axis_success(self):
         fft.irfftn(a, axes=axes)
 
 
+class TestFFTCache(TestCase):
+
+    def test_basic_behaviour(self):
+        c = _FFTCache(max_size_in_mb=1)
+        # Setting
+        c[1] = [np.ones(2)]
+        c[2] = [np.zeros(2)]
+        # Getting
+        assert_array_almost_equal(c[1][0], np.ones(2))
+        assert_array_almost_equal(c[2][0], np.zeros(2))
+        # Setdefault
+        c.setdefault(1, [np.array([1, 2])])
+        assert_array_almost_equal(c[1][0], np.ones(2))
+        c.setdefault(3, [np.array([1, 2])])
+        assert_array_almost_equal(c[3][0], np.array([1, 2]))
+
+        assert c._get_size() == np.ones(2).nbytes * 3
```

Comment:
Interesting point. For pytest and nose I usually just use plain asserts but your point is very valid. I changed it to the `assertEqual()` method of the test class. I hope that's ok.